harvest/harvest.py

```diff
@@ -20,17 +12,21 @@ class HarvestError(Exception):
 
 
 class Harvest(object):
-    def __init__(self, uri, email=None, password=None, client_id=None, token=None, put_auth_in_header=True):
+    def __init__(self, uri, email=None, password=None, refresh_token=None, client_id=None, token=None, put_auth_in_header=True):
         self.__uri = uri.rstrip('/')
         parsed = urlparse(uri)
         if not (parsed.scheme and parsed.netloc):
             raise HarvestError('Invalid harvest uri "{0}".'.format(uri))
-
-        self.__headers = {
-            'Content-Type'  : 'application/json',
-            'Accept'        : 'application/json',
-            'User-Agent'    : 'Mozilla/5.0',  # 'TimeTracker for Linux' -- ++ << >>
```

Comment:
It is not missing. It checks whether it contains refresh_token, else it uses the normal self._headers. I did remove the 'User-Agent': ... but added back for consistency.